eea/reports/browser/displays.py

```diff
@@ -97,7 +97,7 @@ def items(self):
         """ Files children
         """
         return self.context.getFolderContents({
-            'portal_type': ['Document', 'File', 'Link', 'Folder']
+            'portal_type': ['Document', 'File', 'Link', 'Folder', 'Fiche']
```

Comment:
@alecghica  Just get all of the content without any predefined content type, this way we can modify the restricted items
and they will show up.